src/components/Picker/BasePicker/index.js

```diff
@@ -1,30 +1,49 @@
 import React from 'react';
 import RNPickerSelect from 'react-native-picker-select';
+import _ from 'underscore';
 
 import styles from '../../../styles/styles';
 import * as basePickerPropTypes from './basePickerPropTypes';
 import basePickerStyles from './basePickerStyles';
 
-const BasePicker = props => (
-    <RNPickerSelect
-        onValueChange={props.onChange}
-        items={props.items}
-        style={props.size === 'normal' ? basePickerStyles(props.disabled, props.hasError, props.focused) : styles.pickerSmall}
-        useNativeAndroidPickerStyle={false}
-        placeholder={props.placeholder}
-        value={props.value}
-        Icon={() => props.icon(props.size)}
-        disabled={props.disabled}
-        fixAndroidTouchableBug
-        onOpen={props.onOpen}
-        onClose={props.onClose}
-        pickerProps={{
-            onFocus: props.onOpen,
-            onBlur: props.onClose,
-        }}
-    />
-);
+class BasePicker extends React.Component {
+    constructor(props) {
+        super(props);
 
+        this.state = {
+            selectedValue: this.props.value || this.props.defaultValue,
+        };
+    }
+
+    handleChange = (value) => {
```

Comment:
> use the function myFunction keyword rather than const myFunction

Reference: https://github.com/Expensify/App/blob/main/STYLE.md#functions-1

Reply:
Nice catch!

Reply:
Also I found I can not use `handle` world https://github.com/Expensify/App/blob/main/STYLE.md#event-handlers, I'll fix it accordingly.